Tools/cases_generator/analyzer.py

```diff
@@ -137,13 +137,14 @@ class StackItem:
     name: str
     type: str | None
     size: str
+    attributes: list[StackAttribute]
```

Comment:
How about adding `field(default_factory=list)` to avoid having to pass an empty list when instantiating?